### PR TITLE
Tracy pilcher

### PR DESCRIPTION
**Pull request by motorcitycode**
Completed mvp tasks for today.

### CODE REVIEW

**Comment by motorcitycode** [comment]
Hi @eeevvaaa

Done with this one, moving on to attempt the sort thingie.

Thanks,
Tracy

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# React-Todo
+# React-Todo :)
```
Comment (eeevvaaa) [inline README.md]: 😃

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,113 @@
...
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+
+  updateList = event => {
+    event.preventDefault();
+    // Note: prevState returns current state when invoked, unlike this.state.
+    // this.state can change during update.
+    this.setState(prevState => {
```
Comment (eeevvaaa) [inline src/App.js]: Love the comments here!!

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,113 @@
...
+  handleChanges = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+
+  updateList = event => {
+    event.preventDefault();
+    // Note: prevState returns current state when invoked, unlike this.state.
+    // this.state can change during update.
```
Comment (eeevvaaa) [inline src/App.js]: Love the comments here and the use of prevState!

File: src/components/TodoComponents/Todo.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Todo.css'
+
+export default function Todo(props) {
+    let completed = props.item.completed ? ' completed' : '';
```
Comment (eeevvaaa) [inline src/components/TodoComponents/Todo.js]: Great use of the ternary operator!

**Comment by motorcitycode** [comment]
Thank you, these were all techniques from Lambda's lectures and the TK but
I'm glad to understand them and be able to utilize them.

Tracy

**Pull request merged**